torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -395,11 +421,9 @@ def _test_graph(self, fn, exec_mode):
                 ).to_here()
             else:
                 raise ValueError("Unrecognized ExecMode {}".format(exec_mode))
-
```

Comment:
shall we revert unintentionally deleted lines?

Reply:
I deleted these lines on purpose when going through the tests that I updated. I see no reason for having them. Do you want me to add them back?